packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableInserter.ts

```diff
@@ -108,7 +108,7 @@ function getInsertElementData(
     backgroundColor: string
 ): CreateElementData {
     const inserterColor = isDark ? INSERTER_COLOR_DARK_MODE : INSERTER_COLOR;
-    const outerDivStyle = `position: fixed; width: ${INSERTER_SIDE_LENGTH}px; height: ${INSERTER_SIDE_LENGTH}px; font-size: 16px; color: ${inserterColor}; line-height: 10px; vertical-align: middle; text-align: center; cursor: pointer; border: solid ${INSERTER_BORDER_SIZE}px ${inserterColor}; border-radius: 50%; background-color: ${backgroundColor}`;
```

Comment:
is it possible to use css like align: center to style the plus sign, so we do not need to relay on hard coded number?

Reply:
The way the handle is constructed mostly uses hard coded values. We already use vertical and horizontal align to center it, however that is still too low, so we use line-height to correct it further.